.changie.yaml

```diff
@@ -0,0 +1,95 @@
+changesDir: .changes
+unreleasedDir: unreleased
+headerPath: ./header.tpl.md
+versionExt: md
+versionFormat: '## {{.Version}} - {{.Time.Format "2006-01-02"}}'
+kindFormat: '### {{.Kind}}'
+changeFormat: '- {{.Body}} #{{.Custom.Issue}}'
+fragmentFileFormat: '{{.Project}}-{{.Custom.Issue}}-{{.Time.Format "2006-01-02"}}'
+projects:
+  - label: acid
+    key: acid
+    changelog: modules/acid/CHANGELOG.md
+  - label: agar
+    key: agar
+    changelog: modules/agar/CHANGELOG.md
+  - label: alloy
+    key: alloy
+    changelog: modules/alloy/CHANGELOG.md
+  - label: boss
+    key: boss
+    changelog: modules/boss/CHANGELOG.md
+  - label: boulder
+    key: boulder
+    changelog: modules/boulder/CHANGELOG.md
+  - label: bridge
+    key: bridge
+    changelog: modules/bridge/CHANGELOG.md
+  - label: darwin
+    key: darwin
+    changelog: modules/darwin/CHANGELOG.md
+  - label: dragster
+    key: dragster
+    changelog: modules/dragster/CHANGELOG.md
+  - label: jax
+    key: jax
+    changelog: modules/jax/CHANGELOG.md
+  - label: katamari
+    key: katamari
+    changelog: modules/katamari/CHANGELOG.md
+  - label: katamari-assertions
+    key: katamari-assertions
+    changelog: modules/katamari-assertions/CHANGELOG.md
+  - label: mcagar
+    key: mcagar
+    changelog: modules/mcagar/CHANGELOG.md
+  - label: phoenix
+    key: phoenix
+    changelog: modules/phoenix/CHANGELOG.md
+  - label: polaris
+    key: polaris
+    changelog: modules/polaris/CHANGELOG.md
+  - label: porkbun
+    key: porkbun
+    changelog: modules/porkbun/CHANGELOG.md
+  - label: robin
+    key: robin
+    changelog: modules/robin/CHANGELOG.md
+  - label: sand
+    key: sand
+    changelog: modules/sand/CHANGELOG.md
+  - label: snooker
+    key: snooker
+    changelog: modules/snooker/CHANGELOG.md
+  - label: sugar
+    key: sugar
+    changelog: modules/sugar/CHANGELOG.md
+  - label: tinymce
+    key: tinymce
+    changelog: modules/tinymce/CHANGELOG.md
+kinds:
```

Comment:
I think the order of the kinds might be wrong here:
https://www.notion.so/tinycloud/Changelog-Standard-3027cf188f104370984b291578b673d3

I think Improved should be before `Fixed`

Reply:
Ah I thought this was just for the `changie new` prompt. But you're right, this is the order the kinds are emitted into the changelog file as well.

@Skylite73 looks like you copied our list for the repositories you used changie on, and we were wrong...